src/Microsoft.OData.Edm/Vocabularies/CommunityVocabularyConstants.cs

```diff
@@ -14,20 +14,20 @@ public static class CommunityVocabularyConstants
         /// <summary>OData.Community.AlternateKeys.V1.AlternateKeys </summary>
         public const string AlternateKeys = "OData.Community.AlternateKeys.V1.AlternateKeys";
 
-        /// <summary>OData.Community.AlternateKeys.V1.AlternateKey </summary>
-        public const string AlternateKeyType = "OData.Community.AlternateKeys.V1.AlternateKey";
-
         /// <summary>OData.Community.AlternateKeys.V1.AlternateKey.Key </summary>
-        public const string AlternateKeyTypeKeyPropertyName = "Key";
-
-        /// <summary>OData.Community.AlternateKeys.V1.PropertyRef </summary>
-        public const string PropertyRefType = "OData.Community.AlternateKeys.V1.PropertyRef";
+        internal const string AlternateKeyTypeKeyPropertyName = "Key";
 
         /// <summary>OData.Community.AlternateKeys.V1.PropertyRef.Name </summary>
-        public const string PropertyRefTypeNamePropertyName = "Name";
+        internal const string PropertyRefTypeNamePropertyName = "Name";
 
         /// <summary>OData.Community.AlternateKeys.V1.PropertyRef.Alias </summary>
-        public const string PropertyRefTypeAliasPropertyName = "Alias";
+        internal const string PropertyRefTypeAliasPropertyName = "Alias";
+
+        /// <summary>OData.Community.AlternateKeys.V1.AlternateKey </summary>

```

Comment:
Nit: Remove the space between `AlternateKey` and `</summary>`.  Inline change `public` to `internal`, if that is the change to this file.